tests/lib/rules/prop-types.js

```diff
@@ -2493,8 +2493,7 @@ ruleTester.run('prop-types', rule, {
     {
       code: `
         interface Props {
-          'aria-label': string // 'undefined' PropType is defined but prop is never used eslint(react/no-unused-prop-types)
-          // 'undefined' PropType is defined but prop is never used eslint(react-redux/no-unused-prop-types)
+          'aria-label': string;
         }
```

Comment:
why does this test case have to change? i realize they're just comments but it's nice when test diffs are only green :-)

Reply:
OK! I will add it back!